backend/tests/test_helper.ts

```diff
@@ -103,8 +115,13 @@ export const postSignUp = async (
   return response;
 };
 
-export const getUsernameFromToken = (token: string): string => {
-  const { username } = jwt.verify(token, getEnvVar("SECRET")) as jwt.JwtPayload;
+export const getValuesFromToken = (
+  token: string
+): { username: string; id: string } => {
+  const { username, id } = jwt.verify(
+    token,
+    getEnvVar("SECRET")
+  ) as jwt.JwtPayload;
 
-  return username;
+  return { username, id };
```

Comment:
Not a must-change but maybe we should rename `id` to `userId` for easier reference 🤔